w08h01/test/pgdp/SimpleGenericsTest.java

```diff
@@ -141,6 +141,22 @@ void intersection2() {
         Collection<ArrayList>[] listArray = new Collection[]{list1, list, list3};
         assertEquals("[1, 2, 4]", Arrays.toString(SimpleGenerics.intersection(listArray).toArray()));
     }
+    @Test
+    @DisplayName("intersection collections array, with one empty collection")
+    void intersection2() {
```

Comment:
There is already a method called `intersection2`.

```suggestion
    void intersectionWithEmptyList() {
```

Maybe this a better method name?